core/test/functional/routes/api/public_api_spec.js

```diff
@@ -474,6 +608,49 @@ describe('Public API', function () {
             });
     });
 
+    it('browse user with count.posts', function (done) {
+        request.get(testUtils.API.getApiQuery('users/?client_id=ghost-admin&client_secret=not_available&include=count.posts&order=count.posts ASC'))
+            .set('Origin', testUtils.API.getURL())
+            .expect('Content-Type', /json/)
+            .expect('Cache-Control', testUtils.cacheRules.private)
+            .expect(200)
+            .end(function (err, res) {
+                if (err) {
+                    return done(err);
+                }
+
+                var jsonResponse = res.body;
+
+                should.exist(jsonResponse.users);
+                jsonResponse.users.should.have.length(6);
+
+                // We don't expose the email address.
+                testUtils.API.checkResponse(jsonResponse.users[0], 'user', ['count'], null, null, {public: true});
+
+                // Each user should have the correct count
+                _.find(jsonResponse.users, {slug:'joe-bloggs'}).count.posts.should.eql(4);
+                _.find(jsonResponse.users, {slug:'contributor'}).count.posts.should.eql(0);
+                _.find(jsonResponse.users, {slug:'slimer-mcectoplasm'}).count.posts.should.eql(0);
+                _.find(jsonResponse.users, {slug:'jimothy-bogendath'}).count.posts.should.eql(0);
+                _.find(jsonResponse.users, {slug: 'smith-wellingsworth'}).count.posts.should.eql(0);
+                _.find(jsonResponse.users, {slug:'ghost'}).count.posts.should.eql(7);
+
+                const ids = jsonResponse.users
+                    .filter(user => (user.slug !== 'ghost'))
```

Comment:
@kirrg001 do you know of a better way to reference 'ghost' user from tests? e.g.: `testUtils.DataGenerator.Content.users.GhostUser` :thinking:

Reply:
you could require the Ghost fixtures?

Reply:
data/schema/fixtures

Reply:
Still confused 🤔 

```
                {
                    "id": "5951f5fca366002ebd5dbef7",
                    "name": "Ghost",
                    "email": "ghost-author@example.com",
                    "status": "active",
                    "roles": ["Author"],
                    "twitter": "tryghost",
                    "facebook": "ghost",
                    "location": "The Internet",
                    "website": "https://ghost.org",
                    "bio": "You can delete this user to remove all the welcome posts",
                    "profile_image": "https://casper.ghost.org/v2.0.0/images/ghost.png"
                }
```

The ghost user lives in the fixtures. And has a static id.
You can require the file and compare the id.

Reply:
Ok. All clear now, I was grepping 'test' directory for this user. Checked and looks like no other test is actually using fixtures other than: https://github.com/TryGhost/Ghost/blob/master/core/test/unit/data/schema/fixtures/utils_spec.js#L9 . Don't know if it would be a good idea introducing such usage here :thinking: What's your opinion on this?

Reply:
i would keep as is for now and continue?